Makefile

```diff
@@ -15,6 +15,8 @@ INSTALLED_COMPONENTS = $(shell $(RUSTUP) component list --installed --toolchain
 INSTALLED_NIGHTLY_COMPONENTS = $(shell $(RUSTUP) component list --installed --toolchain $(NIGHTLY_TOOLCHAIN))
 AVAILABLE_CARGO_COMMANDS = $(shell $(CARGO) --list)
 
+CARGO_TOML_FILES = $(shell ls **/Cargo.toml)
```

Comment:
Actually, I think you shouldn't even need this `ls`. The Makefile should expand the glob directly no?